mod-orders/schemas/composite_purchase_order.json

```diff
@@ -109,6 +109,14 @@
         "$ref": "composite_po_line.json"
       }
     },
+    "units": {
```

Comment:
Please append "Ids" to the end of the name of field which to highlight that this array contains UUIDs i.e. `unitIds` or `acqUnitIds`.